pkg/client/tools.go

```diff
@@ -123,11 +166,97 @@ func ImageImportIntoClusterMulti(ctx context.Context, runtime runtimes.Runtime,
 			l.Log().Errorf("failed to delete tools node '%s' (try to delete it manually): %v", toolsNode.Name, err)
 		}
 	}
+	return nil
+}
 
-	l.Log().Infoln("Successfully imported image(s)")
+func importWithStream(ctx context.Context, runtime runtimes.Runtime, imagesFromRuntime []string, cluster *k3d.Cluster, imagesFromTar []string) error {
```

Comment:
I think the parameters here should have the same order as in `importWithToolsNode` :+1: